SourceX/storm/storm_net.cpp

```diff
@@ -69,6 +71,19 @@ BOOL SNetDropPlayer(int playerid, DWORD flags)
 
 BOOL SNetGetGameInfo(int type, void *dst, unsigned int length, unsigned int *byteswritten)
 {
+	switch (type) {
+	case GAMEINFO_NAME:
+		strncpy((char *)dst, gpszGameName, length);

```

Comment:
I probably would use `snprintf` instead as `strncpy` doesn't null terminate the string so a user providing a 128 char name might be able to break the client.